apps/session-manager/src/services/authentication-lock.ts

```diff
@@ -0,0 +1,19 @@
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/TaskEither";
+import * as ROA from "fp-ts/ReadonlyArray";
+import { LockedProfileRepo } from "../repositories";
+import { LockUserAuthenticationDeps } from "../repositories/locked-profile";
+
+type AuthLockServiceDeps = LockUserAuthenticationDeps;
+
+export const isUserAuthenticationLocked: (
+  fiscalCode: FiscalCode,
+) => RTE.ReaderTaskEither<AuthLockServiceDeps, Error, boolean> =
+  (fiscalCode) => (deps) =>
+    pipe(
+      deps,
+      LockedProfileRepo.getUserAuthenticationLocks(fiscalCode),
+      TE.map(ROA.isNonEmpty),
+    );
```

Comment:
We can simply this way

```suggestion
export const isUserAuthenticationLocked: (
  fiscalCode: FiscalCode,
) => RTE.ReaderTaskEither<AuthLockServiceDeps, Error, boolean> = (fiscalCode) =>
  pipe(
    LockedProfileRepo.getUserAuthenticationLocks(fiscalCode),
    RTE.map(ROA.isNonEmpty),
  );
```

Reply:
Fixed in 2bbd7e8